acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/utils/MetadataFileHelpers.java

```diff
@@ -184,49 +179,20 @@ public void createAzureKeyYamlFileAt(
     }
   }
 
-  public void createYubiHsmYamlFileAt(
-      final Path metadataFilePath,
-      final Path destinationYubiShellSimulator,
-      final KeyType keyType) {
+  public void createYubiHsmYamlFileAt(final Path metadataFilePath, final KeyType keyType) {
     final int opaqueObjId = keyType == KeyType.BLS ? 1 : 2;
 
-    final Map<String, Serializable> yaml =
-        Map.of(
-            "type",
-            "yubihsm2",
-            "yubiShellBinaryPath",
-            destinationYubiShellSimulator.toString(),
-            "connectorUrl",
-            "http://localhost:12345",
-            "authKey",
-            1,
-            "password",
-            "password",
-            "opaqueObjId",
-            opaqueObjId,
-            "keyType",
-            keyType.name());
+    final Map<String, String> yaml = new HashMap<>();
```

Comment:
nit: happy for this to stay as serializable if that makes life easier

Reply:
Its all good .... the YAML format doesn't use quotes for strings/numbers (so we are good).